docs/run-with-docker.md

```diff
@@ -6,7 +6,7 @@ In case you have a lot of trouble installing polyml and HOL, you can use Docker
 
 1. Create a `Dockerfile` in the root of the project with the following content:
 
-```Dockerfile
```

Comment:
why this extra whitespace?

Reply:
whoops! https://github.com/verifereum/verifereum/pull/33/commits/033e3dedf181165093c9aab69e28df60a79e85bc